src/clients/inst_kickoff.rb

```diff
@@ -44,8 +44,8 @@ def main
           # When upgrading system, remove devs.rpm if installed
```

Comment:
Hm, this has been added 12 years ago, the `devs` package has been dropped long time ago. https://maintenance.suse.de/maintained/?package=devs displays SLE10.

So I'd rather completely remove this obsolete code.

Reply:
ok, I do not know it.